index.html

```diff
@@ -52,6 +52,11 @@ <h2 class="header__subtitle">
             d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z" />
         </svg>
         <input id="search-input" type="text" name="search-input" placeholder="Pesquisar..." class="searchbar__input" />
+        <button type="button" id="clear-input">
+          <svg xmlns="http://www.w3.org/2000/svg" class="clearinput__icon" x="0px" y="0px" width="100" height="100" viewBox="0 0 50 50">
```

Comment:
Para o nome da classe, seguindo o padrão BEM, que tal se colocarmos o nome da classe como "searchbar__clear"?

```suggestion
          <svg xmlns="http://www.w3.org/2000/svg" class="searchbar__clear" x="0px" y="0px" width="100" height="100" viewBox="0 0 50 50">
```